### DISCUSSION TITLE
Как прописать Кластер №2,№3,№4,....,№n ?

### DISCUSSION THREAD

**Discussion by KEJUDIC** [MrZillaGold/VK2Discord#760] [Q&A] [answered]
Всем доброго времени суток, вот столкнулся с такой задачей, и нигде не могу найти ответ, сам конечно догадываюсь что просто скопировать Кластер №1, и поменять ID- группы, но я решил уточнить, что бы быть уверенным, что делаю.
То есть, если я хочу что бы бот просматривал сразу насколько групп, и с разных пабликов брал новости.
Мне нужно создать второй кластер , да ? 
Если да то как это сделать, если нет, то какие есть варианты ?

**Comment by egorprnn** [comment, marked as answer]
Пример с двумя кластерами есть в репозитории
https://github.com/MrZillaGold/VK2Discord/blob/master/config_example.json
